app/routes/wow.ultrarare._index.tsx

```
@@ -0,0 +1,1277 @@
+                        type="button"
+                        onClick={() => removeFilter(filter.id)}
+                        className="px-2 py-1 bg-red-500 text-white rounded hover:bg-red-600 transition-colors text-xs ml-2 flex-shrink-0">
+                        Remove
+                      </button>
+                    </div>
+                    <div className="flex flex-wrap gap-4 items-end">
+                      <div className="w-64">
+                        <label className="block text-sm font-medium text-gray-700 dark:text-gray-100 mb-1">
+                          Column
+                        </label>
+                        <select
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)